examples/asynch_adaptive_msm/async_iteration.py

```diff
@@ -72,9 +84,18 @@ async def simulation_iteration(inputs: Iterable[Future[SimulationInput]]):
 
 async def simulation_and_analysis_loop(initial_input):
     simulation_input = initial_input
+    next_conformation = simulation_input.conformation
     model = msmtool.msm_analyzer()
     while not model.is_converged():
-        model, simulation_input = analysis_iteration(model, simulation_iteration(simulation_input))
+        # Option A would combine phase 0 and 1. E.g.:
+        #     simulation = simulation_iteration_optionA(simulation_input, next_conformation)
+        # Option B:
+        # Phase 0: Prepare simulation input with the next round of initial conformations.
+        simulation_input = modify_input(simulation_input, conformation=next_conformation)
```

Comment:
> Would this be clearer

It seems unnecessarily verbose. To my taste, the referent of `simulation_input` is determined by the preceding sequence of code (the larger context) and the name doesn't need to add such specificity. But I would not object to such a change, if you think it would improve readability.

> seems to be setting up modify input

It will be a long process, I'm sure, but I think we can and should be trying to refine the names and concepts for the SimulationInput components, or compartmentalized notions of the information in a SimulationInput.

Key words that are missing from the `modify_input` are assumed not to be modified. (In general, we probably don't need to know whether any particular parameter exists at all.)

I don't know if you are looking for a distinction I am making between a molecular conformation and an atomic configuration, but I have not attempted to do so.